DuckDuckGo/NetworkProtectionDebugViewController.swift

```diff
@@ -191,13 +191,13 @@ final class NetworkProtectionDebugViewController: UITableViewController {
 
     private func configure(_ cell: UITableViewCell, forDebugFeatureAtRow row: Int) {
         switch DebugFeatureRows(rawValue: row) {
-        case .enableAlwaysOn:
-            cell.textLabel?.text = "Enable Always On"
+        case .toggleAlwaysOn:
+            cell.textLabel?.text = "Always On"
```

Comment:
Changed the name since the tick in the debug menu options indicates whether this is ON or OFF.